### PR TITLE
Verbose SSL error logs

### PR DESCRIPTION
**Pull request by serrislew**
Adding more verbose information to SSL error and debug logs
- Explicitly stating (i.e. "[empty file]") for empty files to avoid confusion 

Including #9222 with slight modifications

### CODE REVIEW

**Comment by maskit** [comment]
[approve ci fedora]

File: iocore/net/SSLSecret.cc [hunk 1]
```diff
@@ -55,12 +55,14 @@ SSLSecret::loadFile(const std::string &name, std::string &data_item)
   struct stat statdata;
   // Load the secret and add it to the map
   if (stat(name.c_str(), &statdata) < 0) {
+    Debug("ssl_secret", "File does not exist or cannot be found: %s", name.c_str());
```
Comment (mlibbey) [inline iocore/net/SSLSecret.cc:58]: Perhaps a bike shed comment... I'm struggling to think of the difference between doesn't exist and "cannot be found". Would "cannot be read" (for instance a file ownership or permissions issue) describe an issue caught in this if statement?

File: iocore/net/SSLUtils.cc [hunk 2]
```diff
@@ -2222,16 +2233,19 @@ SSLMultiCertConfigLoader::load_certs_and_cross_reference_names(
     key_tok.setString("");
   }
 
-  if (sslMultCertSettings && sslMultCertSettings->key && cert_tok.getNumTokensRemaining() != key_tok.getNumTokensRemaining()) {
-    Error("the number of certificates in ssl_cert_name and ssl_key_name doesn't match");
+  size_t cert_tok_num = cert_tok.getNumTokensRemaining();
+  if (sslMultCertSettings && sslMultCertSettings->key && cert_tok_num != key_tok.getNumTokensRemaining()) {
+    Error("the number of certificates in ssl_cert_name (%zu) and ssl_key_name (%zu) doesn't match", cert_tok_num,
```
Comment (mlibbey) [inline iocore/net/SSLUtils.cc:2238]: Can we change "doesn't" to "do not" (think the latter is better grammar).

File: iocore/net/SSLUtils.cc [hunk 3]
```diff
@@ -2222,16 +2233,19 @@ SSLMultiCertConfigLoader::load_certs_and_cross_reference_names(
...
+          key_tok.getNumTokensRemaining());
     return false;
   }
 
   SimpleTokenizer ca_tok("", SSL_CERT_SEPARATE_DELIM);
   if (sslMultCertSettings && sslMultCertSettings->ca) {
     ca_tok.setString(sslMultCertSettings->ca);
     if (cert_tok.getNumTokensRemaining() != ca_tok.getNumTokensRemaining()) {
-      Error("the number of certificates in ssl_cert_name and ssl_ca_name doesn't match");
+      Error("the number of certificates in ssl_cert_name (%zu) and ssl_ca_name (%zu) doesn't match", cert_tok_num,
```
Comment (mlibbey) [inline iocore/net/SSLUtils.cc:2247]: doesn't --> do not

**Comment by maskit** [comment]
Just rerunning the test on Fedora worked.

**Review by ywkaras — APPROVED** [review]

**Comment by zwoop** [comment]
Cherry-picked to v9.2.x

**Pull request merged**